C/CDE/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+using BinaryBuilder
+
+name = "CDE"
+version = v"1.0.0"
+sources = [GitSource("https://github.com/HabershonLab/cde", "049cd0d2da73a0c5bb340ebdb9a78e35348fe49a")]
+
+script = raw"""
+if [[ ${target} == *w64* ]]; then
+    exe_out="${bindir}/cde.exe"
+else
+    exe_out="${bindir}/cde"
+fi
```

Comment:
`exeext` is our custom env variable for the extension of executables:
```suggestion
exe_out="${bindir}/cde${exeext}"
```

Reply:
Ah thanks, that's exactly what I was looking for!